src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Data
+public class BedAssignment {
```

Comment:
Are you going to add some Swagger docs/annotations?

Reply:
That's a good idea, will do in a following PR, I know we have more things in the pipeline around this.